fix gh build action error 3

@claude when i run the .gitbub/workflows/build-app.yml it is failing with this error:
```
Run npm run typecheck

> openwebui-config-tool@1.0.0 typecheck
> tsc --noEmit


> openwebui-config-tool@1.0.0 lint
> eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0


Oops! Something went wrong! :(

ESLint: 8.57.1

ESLint couldn't find the config "@typescript-eslint/recommended" to extend from. Please check that the name of the config is correct.

The config "@typescript-eslint/recommended" was referenced from the config file in "/home/runner/work/openwebui-config.com/openwebui-config.com/.eslintrc.cjs".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

Error: Process completed with exit code 2.

```

help me resolve this

--- a/.eslintrc.cjs
+++ b/.eslintrc.cjs
@@ -3,12 +3,12 @@ module.exports = {
   env: { browser: true, es2020: true },
   extends: [
     'eslint:recommended',
-    '@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended',
     'plugin:react-hooks/recommended',
   ],
   ignorePatterns: ['dist', '.eslintrc.cjs'],
   parser: '@typescript-eslint/parser',
-  plugins: ['react-refresh'],
+  plugins: ['react-refresh', '@typescript-eslint'],
   rules: {
     'react-refresh/only-export-components': [
       'warn',

